geoapi/routes/projects.py

```diff
@@ -169,14 +169,24 @@ def get(self):
         uuid_subset = query.get("uuid")
 
         if uuid_subset:
-            logger.info("Get a subset of projects for user:{} projects:{}".format(u.username, uuid_subset))
+            logger.info(f"Getting a subset of projects for user:{u.username} projects:{uuid_subset}")
```

Comment:
It looks like the uuid_subset never exists for a Taggit get request. I see the header X-Geoapi-Application is set to taggit in the response, but since uuid_subset is None, it never fires the log. I put a debugging statement for uuid_subset and clicking around to where I would expect that logging event to fire, but it returns as None.

Reply:
Okay, i'm leaning towards moving this to the features route.  


----------


🎗️ I'll also update the statements to match what we decide here:  https://github.com/TACC-Cloud/geoapi/pull/163#discussion_r1410921617

Reply:
We also need to look logging which version of the map is being accessed. i.e. the public or non-public version 🤔